core/state/state_object.go

```diff
@@ -130,7 +122,7 @@ func newObject(db *StateDB, address common.Address, data Account) *stateObject {
 
 // EncodeRLP implements rlp.Encoder.
 func (s *stateObject) EncodeRLP(w io.Writer) error {
-	return rlp.Encode(w, s.data)
+	return rlp.Encode(w, &s.data)
```

Comment:
This changes.. something? Not sure if it makes any difference -- but is it safe?

Reply:
it won't make any difference because this `EncodeRLP` is not called anymore. I don't think that it will hurt in any case, and would save a copy when called.